src/components/SignupConfirmation.tsx

```diff
@@ -0,0 +1,161 @@
+                </Form.Item>
+
+                <Form.Item
+                    name="check2"
+                    rules={[{ required: true, message: 'Please check all boxes' }]}
+                    style={{ display: 'inline-block'}}
+                >
+                    <Checkbox>
+                        All parents are to remain at The Sajni Center during programs
+                    </Checkbox>
+                </Form.Item>
+
+                <Form.Item
+                    name="check3"
+                    rules={[{ required: true, message: 'Please check all boxes' }]}
+                    style={{ display: 'inline-block'}}
+                >
+                    <Checkbox>
+                        My children are up to date on all vaccinations and I will provide a copy of my children’s
+                        immunization records prior to attending any programs. (Children who are being treated for cancer
+                        often have severely compromised immune systems, so we are required to collect this
+                        information for their safety. All medical information will be stored in a HIPPA-compliant manner.)
+                    </Checkbox>
+                </Form.Item>
+            </div>
+
+            <div style={{marginTop: '36px',
+                    fontSize: '18px',}}>
+                <p className="photoRelease">
+                    <strong>Photo/Video Release</strong><br/>
+                    Do you authorize the use and reproduction by Lucy’s Love Bus of any and all photographs
+                    and any other audio-visual materials taken of me for promotional material, educational
+                    activities, exhibitions or for any other use for the benefit of the organization?
+                    <br/>
+                    <br/>
+                    <i> <strong> We do not share last names, diagnosis, or hometowns unless given explicit permission in order to protect privacy</strong></i>
+                </p>
+
+                <Form.Item name="photoRelease"
+                           rules={[{ required: true, message: 'Please select one option.'}]}>
+                    <Radio.Group className="photo-release-radio">
+                        <Radio value={1} style={radioStyle}>
+                            I consent to photo/video release
+                        </Radio>
+                        <Radio value={2} style={radioStyle}>
+                            I do not consent to photo/video release
+                        </Radio>
+                    </Radio.Group>
+                </Form.Item>
+            </div>
+
+            <div>
+                <Form.Item
+                    label="Head of Family Name"
+                    name="headOfFamilyName"
+                    rules={[{ required: true, message: 'Please input the name of the head of the family' }]}
+                    style={{ display: 'inline-block', width: 'calc(50% - 8px)', margin: '0 8px 0px 0px' }}
```

Comment:
Same as above, please put this in a LESS file if possible! Inline CSS should be avoided here